tasks/redhat.yml

```diff
@@ -14,6 +14,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+- name: yum install gnupg2
+  yum:
+    name: "gnupg2"
+    state: "present"
+  register: result
+  until: result is succeeded
+  tags: gnucash
+
+- name: yum-config-manager --add-repo
+  yum:
+    name: "{{ item.name }}"
+    state: "{{ item.state }}"
+    enablerepo: "{{ item.enablerepo }}"
```

Comment:
`enablerepo` should go to below `yum install` section... NOT here for installing EPEL repo rpm...

Reply:
oh yes, I have also changed this part.
tasks/redhat:
```yml
- name: yum install
  yum:
    name: "{{ item.name }}"
    state: "{{ item.state }}"
    enablerepo: "{{ item.enablerepo }}"
  loop: "{{ _yum }}"
  register: result
  until: result is succeeded
  tags: gnucash
```
Sorry, I forgot to mention this part.

togerther with
vars/centos-8
```yml
_rpm_repository:
  - name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
    state: "latest"

_yum:
  - { state: "latest", name: "gnucash", enablerepo: "epel-testing"}
```
Therefore, as mentioned below, an error message `"No package gnucash available."` pop out.